hazelcast/src/main/java/com/hazelcast/query/impl/predicates/PredicateDataSerializerHook.java

```diff
@@ -152,6 +152,11 @@ public IdentifiedDataSerializable createNew(Integer arg) {
                 return new PartitionPredicateImpl();
             }
         };
+        constructors[PARTITIONS_PREDICATE] = new ConstructorFunction<Integer, IdentifiedDataSerializable>() {
```

Comment:
we don't need this change, previous `PARTITION_PREDICATE ` should be sufficient.

Reply:
Good catch. A leftover from the prior implementation attempt.